ddangcoonfire/experiment-scheduler
# Ddan 118 execute filesoption

### PURPOSE OF PR
* Simple summary of this PR

Add "files" option to yaml file which is used in "exs execute -f <yaml>".
The file option contains a list of file names to be uploaded before proceeding the experiments in the task manager.

### What type of PR is it?
* fix / feature / improvement /  hot-fix / refactoring

feature

### JIRA ISSUE
* https://ddangcoonfire.atlassian.net/browse/DDAN-118


### HAVE YOU CHECKED ...
* [x] ADD JIRA ID ON PR TITLE (ex. [DDAN-1234] PR TITLE)
* [x] pre-commit
* [x] Branch Format ( type/issue-NUM ) (ex. refactoring/DDAN-1234 )


### Explanation
( Adding screenshot would be great help )
![image](https://github.com/ddangcoonfire/experiment-scheduler/assets/35250512/61baeab6-2e7e-47a0-ab7e-b53ccdcfa750)
![image](https://github.com/ddangcoonfire/experiment-scheduler/assets/35250512/a16b7ac0-bad4-4fe4-a4d0-a8dfeeac77b8)
![image](https://github.com/ddangcoonfire/experiment-scheduler/assets/35250512/3d704ef8-4d19-447b-810e-167e9ec9fa2a)
![image](https://github.com/ddangcoonfire/experiment-scheduler/assets/35250512/5e697370-925f-4b72-b7fd-b257d1edf72b)

Before executing experiments in task manager, we need to upload required files. So we add "files" option in yaml file.
It makes submitter to upload all files in the file list before execute experiments.


### How to Test (Optional)
1) write the python file somewhere you want to. (recommend to make in different directory of exs)
2) add files option to yaml file, and give list with file_path as input
3) execute with the yaml file
4) the file should be downloaded to the exs directory. 

### Question (Optional)



### Etc (Optional)
